plugins/local_file/src/main.hpp

```diff
@@ -0,0 +1,35 @@
+#include "interface.h"
+
+#include <optional>
+#include <shared_mutex>
+#include <string>
+#include <vector>
+
+#include <nlohmann/json.hpp>
+
+#ifndef USER_MAPPER_CPP
+#define USER_MAPPER_CPP
```

Comment:
The header includes need to go inside the include guard.

Reply:
Done.